Assets/HoloToolkit/UX/Resources/Models/Cursor/Cursor_move/CursorRing.fbm/templates.meta

```diff
@@ -1,8 +1,8 @@
 fileFormatVersion: 2
-guid: 20978cf8cb27a50459a532d464f4fece
+guid: ea00e5b153ad7d949b552142891f6c3d
```

Comment:
Is changing the reference to this file going to break anything?

Reply:
Possibly. I'll revert the change. Better safe than sorry.

Reply:
This is another thing that should be cleaned up in the model itself. I believe we used these exact models in the Academy courses, where we saw the same thing.

Basically, the fbx generates an empty folder called templates when imported into Unity, which git ignores because it's empty, then Unity deletes the meta file because the folder doesn't exist, then it recreates the empty folder and the meta when it reloads the model in some cases.